src/index.html

```diff
@@ -6,9 +6,39 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__wrap">
+          <a href="#" class="logo"><img src="./images/logo.png" alt="moyo"></a>
+          <nav>
+            <ul class="nav-list">
+              <li class="nav-list__item"><a href="/" class="nav-list__link is-active">Apple</a></li>
+              <li class="nav-list__item"><a href="#" class="nav-list__link">Samsung</a></li>
+              <li class="nav-list__item"><a href="#" class="nav-list__link">Smartphones</a></li>
```

Comment:
Check out the html formatting guide [here](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md).

```suggestion
            <ul class="nav-list">
              <li class="nav-list__item">
                <a href="/" class="nav-list__link is-active">Apple</a>
              </li>

              <li class="nav-list__item">
                <a href="#" class="nav-list__link">Samsung</a>
              </li>

              <li class="nav-list__item">
                <a href="#" class="nav-list__link">Smartphones</a>
              </li>

```